benchmarks/CMakeLists.txt

```diff
@@ -29,6 +29,12 @@ else()
   target_compile_options(benchmark_main PRIVATE -w)
 ENDIF()
 
+option(ARBORX_ENABLE_CALIPER "Enable Caliper instrumentation" OFF)
+if(ARBORX_ENABLE_CALIPER)
+  find_package(caliper REQUIRED)
```

Comment:
Maybe I should add `FetchContent_Declar` and build caliper automatically if not found, similarly to benchmark. Would be easier for a user.

Reply:
If we use it for a single benchmark, then I don't think it's worth to build caliper automatically